test/e2e/atlas/streams_test.go

```diff
@@ -222,6 +222,31 @@ func TestStreams(t *testing.T) {
 		a.Equal("test-namespace.servicebus.windows.net", *privateLinkEndpoint.DnsDomain)
 	})
 
+	t.Run("List all streams privateLink endpoints", func(t *testing.T) {
+		streamsCmd := exec.Command(cliPath,
+			"streams",
+			"privateLinks",
```

Comment:
[q/nit/just-asking] Do we want to standardize on `"privateLink"` or `"privateLinks"` in these tests? I don't think it matters but I'm not sure if it's mentioned in a doc or wiki

Reply:
I chose to switch it up here mainly because this command reads as something that would return multiple privateLinks so I just thought it kind of "flowed". Not my motivation at the time, but I could see an argument to have at least one test call through the plural just to ensure that the command never changes or that the aliases (singular or plural) do in fact work.

Reply:
Sounds good to me